server/@types/curiousApiClient/index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'curiousApiClient' {
+  import { components } from '../curiousApi'
+
+  export type LearnerEducation = components['schemas']['LearnerEducationDTO']
+  export type LearnerNeurodivergence = components['schemas']['LearnerNeurodivergenceDTO']
```

Comment:
There are other types in the Curious API schema that we will need as we get into more stories that integrate with Curious; but this is sufficient for now.  (In fact all we really need for this story is `LearnerNeurodivergenceDTO` but I also added `LearnerEducationDTO` as we will need that for subsequent stories)